### DISCUSSION TITLE
Is it possible to re-execute the stream load by specifying a previously existing failed label?

### DISCUSSION THREAD

**Discussion by cherry8x** [apache/doris#30201] [A - General / Q&A] [answered]
重新执行，不携带具体数据，只有label信息这样

**Comment by yagagagaga** [comment, marked as answer]
You can retry it with same label and it should be noted that data also needs to be transmitted.
For more information, you can refer
https://doris.apache.org/zh-CN/docs/data-operate/import/import-scenes/load-atomicity#label-%E6%9C%BA%E5%88%B6
